protocol/x/listing/types/errors.go

```diff
@@ -1,5 +1,18 @@
 package types
 
+import errorsmod "cosmossdk.io/errors"
+
 var (
-// Add x/listing specific errors here
+	// Add x/listing specific errors here
+	ErrReferencePriceZero = errorsmod.Register(
+		ModuleName,
+		1,
+		"reference price is zero",
+	)
+
+	ErrReferencePriceOutOfRange = errorsmod.Register(
```

Comment:
nit: it's not reference price out of this range but `p` (im not sure the right term used to describe it)